src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/JsonbGetFieldAsText.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Doctrine\ORM\Query\AST\Functions;
+
+/**
+ * Implementation of PostgreSql json field retrieval as text, filtered by key (using ->>).
+ *
+ * @see https://www.postgresql.org/docs/9.4/static/functions-json.html
+ * @since 0.1
```

Comment:
```suggestion
 * @since 1.8
```
or whatever the next release where this change will be added; 0.1 is a very old release and the annotation is misleading.